converters/toZigbee.js

```diff
@@ -1717,6 +1717,36 @@ const converters = {
             }
         },
     },
+    xiaomi_ZNCZ11LM_power_outage_memory: {
+        key: ['power_outage_memory'],
+        convertSet: async (entity, key, value, meta) => {
+            const payload = value ?
+                [0xaa, 0x80, 0x05, 0xd1, 0x47, 0x00, 0x01, 0x10, 0x01] :
+                [0xaa, 0x80, 0x05, 0xd1, 0x47, 0x01, 0x01, 0x10, 0x00];
+            await entity.write('genBasic', {0xFFF0: {value: payload, type: 0x41}}, manufacturerOptions.xiaomi);
+            return {state: {power_outage_memory: value}};
+        },
+    },
+    xiaomi_ZNCZ11LM_led_disabled_night: {
```

Comment:
Please merge this converter with https://github.com/Koenkk/zigbee-herdsman-converters/blob/0a757bf9787a2cbcaccef27a178beb260966b6a9/converters/toZigbee.js#L1771